.github/workflows/main.yml

```diff
@@ -88,6 +88,9 @@ jobs:
     - name: Install Rust (rustup)
       run: rustup update ${{ matrix.rust }} --no-self-update && rustup default ${{ matrix.rust }}
     - run: rustup target add ${{ matrix.target }}
+    - name: Install Cargo-Wasi
+      if: matrix.target == 'wasm32-wasi'
+      run: cargo install cargo-wasi
```

Comment:
Sorry but what I meant was can this be removed entirely? I don't think `cargo wasi` is needed here

Reply:
I removed cargo-wasi as per your request, seems to be building & running fine. But we lost the auto-`wasm-opt` in release mode. Seems fine to me for a CI.